packages/clay-css/src/scss/cadmin/variables/_range.scss

```diff
@@ -80,6 +80,11 @@ $cadmin-clay-range-input: map-deep-merge(
 			position: absolute,
 			top: 50%,
 			width: 50%,
+			focus: (
```

Comment:
You are right about something wonky going on with `clay-range-variant` or `clay-range-input-variant`. Something might be off with the way we are merging here https://github.com/liferay/clay/blob/dfe8e4d9789e2e2586bd439db2496c9f64388d39/packages/clay-css/src/scss/mixins/_forms.scss#L2287-L2312.

https://github.com/liferay/clay/blob/dfe8e4d9789e2e2586bd439db2496c9f64388d39/packages/clay-css/src/scss/variables/_range.scss#L128-L144 works but https://github.com/liferay/clay/blob/dfe8e4d9789e2e2586bd439db2496c9f64388d39/packages/clay-css/src/scss/atlas/variables/_range.scss#L29-L32 doesn't.

It's weird and convoluted because we are keeping backward compatibility with Clay 2.x.